src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -902,104 +920,72 @@ public void logOut(String wikiURL, String articleName) {
-//
-//      HttpClient httpclient = HttpClientBuilder.create()
-//          .setConnectionBackoffStrategy(new DefaultBackoffStrategy())
-//          .setDefaultCookieStore(cookieStore)
-//          .disableAutomaticRetries()
-//          .build();
-//      HttpPost httpPost = new HttpPost(wikiURL + "wikia.php");
-//      List<NameValuePair> nvps = new ArrayList<NameValuePair>();
-//
-//      nvps.add(new BasicNameValuePair("controller", "UserLoginSpecial"));
-//      nvps.add(new BasicNameValuePair("format", "json"));
-//      nvps.add(new BasicNameValuePair("method", "retrieveLoginToken"));
-//
-//      httpPost.setEntity(new UrlEncodedFormEntity(nvps, StandardCharsets.UTF_8));
-//
-//      HttpResponse response = httpclient.execute(httpPost);
-//      HttpEntity entity = response.getEntity();
-//
-//      JSONObject responseValue = new JSONObject(EntityUtils.toString(entity));
-//
-//      PageObjectLogging.log("LOGIN HEADERS: ", response.toString(), true);
-//      PageObjectLogging.log("LOGIN RESPONSE: ", responseValue.toString(), true);
-//
-//      String token = responseValue.getString("loginToken");
-//
-//      List<NameValuePair> nvps2 = new ArrayList<NameValuePair>();
-//
-//      nvps2.add(new BasicNameValuePair("username", userName));
-//      nvps2.add(new BasicNameValuePair("password", password));
-//      nvps2.add(new BasicNameValuePair("loginToken", token));
-//
-//      HttpPost httpPost4 = new HttpPost(wikiURL + "wiki/Special:UserLogin");
-//      httpPost4.setEntity(new UrlEncodedFormEntity(nvps2,
-//                                                   StandardCharsets.UTF_8));
-//
-//      response = httpclient.execute(httpPost4);
-//
-////      entity = response.getEntity();
-////      for (int i = 0; i < 10; i++) {
-////
-////
-////        xmlResponse = EntityUtils.toString(entity);
-////
-////        xmlResponseArr = xmlResponse.split("\"");
-////
-////        if (xmlResponse.contains("WrongPass")) {
-////          throw new WebDriverException("Incorrect password provided for user: " + userName);
-////        }
-////
-////        if (xmlResponseArr.length >= 11) {
-////          break;
-////        }
-////      }
-//
-//      PageObjectLogging.log("LOGIN HEADERS: ", response.toString(), true);
-////      PageObjectLogging.log("LOGIN RESPONSE: ", xmlResponse, true);
-//
-//      for (org.apache.http.cookie.Cookie cookie : cookieStore.getCookies()) {
-//        System.out.println(cookie.toString());
-//        driver.manage().addCookie(
-//            new Cookie(cookie.getName(), cookie.getValue(), cookie.getDomain(), cookie.getPath(),
-//                       cookie.getExpiryDate(), false));
-//      }
-//
-//      try {
-//        driver.get(wikiURL);
-//      } catch (TimeoutException e) {
-//        PageObjectLogging.log("loginCookie",
-//                              "page timeout after login by cookie", true);
-//      }
-//
+    String client_id = HeliosConfig.getClientId();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)